added online help for parameter ok (#49)
jenkinsci/pipeline-input-step-plugin: README.md

[![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/pipeline-input-step.svg)](https://plugins.jenkins.io/pipeline-input-step)
[![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/pipeline-input-step.svg?color=blue)](https://plugins.jenkins.io/pipeline-input-step)

Adds the Pipeline `input` step to wait for human input or approval. 

> For further understanding,check the [documentation](https://plugins.jenkins.io) for pipeline-input-step plugin.  The  [Pipeline Syntax Snippet Generator](https://www.jenkins.io/redirect/pipeline-snippet-generator) guides the user to select checkout options.

Reviewer: I think it would be better if the first sentence of the paragraph recommended the Pipeline Syntax Snippet Generator and the second sentence directed them to the detailed documentation of the input step.  Directing them to the https://plugins.jenkins.io page causes the reader to open a page with very little content.
```suggestion
Use the  [Pipeline Syntax Snippet Generator](https://www.jenkins.io/redirect/pipeline-snippet-generator) to select options for the `input` step.
For further understanding, check the [pipeline input step plugin documentation](https://www.jenkins.io/doc/pipeline/steps/pipeline-input-step/).
```

> For further understanding,check the https://plugins.jenkins.io[documentation] for pipeline-input-step plugin.  The link:https://www.jenkins.io/redirect/pipeline-snippet-generator [Pipeline Syntax Snippet Generator] guides the user to select checkout options.

Reviewer: Use markdown format instead of asciidoc format for the hyperlink markup.
```suggestion
For further understanding,check the [documentation](https://plugins.jenkins.io) for pipeline-input-step plugin.  The  [Pipeline Syntax Snippet Generator](https://www.jenkins.io/redirect/pipeline-snippet-generator) guides the user to select checkout options.
```

## Version History

> Please refer to [the changelog](CHANGELOG.md) SCM link:https://www.jenkins.io/doc/pipeline/steps/workflow-scm-step/[`checkout` step].

Reviewer: ```suggestion
Please refer to [the changelog](CHANGELOG.md) and SCM [`checkout` step](https://www.jenkins.io/doc/pipeline/steps/workflow-scm-step/).
```

> Please refer to [the changelog](CHANGELOG.md) and SCM [`checkout` step](https://www.jenkins.io/doc/pipeline/steps/workflow-scm-step/).

Reviewer: I recommend that this section be left as it was previously, a reference to the version history, rather than mixing a reference to something that is not related to version history.
```suggestion
Please refer to [the changelog](CHANGELOG.md)
```
